README.md

```diff
@@ -482,26 +449,24 @@ specific CSV, but does not find it (the operator was not present on the cluster
 
 ```shell
 ------------------------------
-operator Runs test on operators when under test is: my-etcd/etcdoperator.v0.9.4  
-  tests for: CSV_SCC
-  /Users/$USER/cnf-cert/test-network-function/test-network-function/operator/suite.go:129
-2021/04/20 14:47:52 Sent: "oc get csv etcdoperator.v0.9.4 -n my-etcd -o json | jq -r 'if .spec.install.spec.clusterPermissions == null then null else . end | if . == null then \"EMPTY\" else .spec.install.spec.clusterPermissions[].rules[].resourceNames end'\n"
+operator Runs test on operators 
+  should eventually be verified as certified (operator redhat-marketplace/etcd-operator)
+  /home/deliedit/redhat/github/david/ssh/test-network-function/test-network-function/operator/suite.go:146
 
-• Failure [10.001 seconds]
```

Comment:
why do we have a report in here ?
I guess this is an error. right ?

Reply:
Same here , this is an example of output similar to what was there before, I just updated with new suites